pandas/tests/indexes/numeric/test_join.py

```diff
@@ -34,7 +34,7 @@ def test_join_inner(self):
         lidx = lidx.take(ind)
         ridx = ridx.take(ind)
 
-        eres = Index([2, 12], dtype=np.int64)
+        eres = Index([2, 12], dtype=np.int64, name="lhs")
```

Comment:
Out of scope here - but the comment on L31 above is no longer true (I think) and we can avoid sorting in this test.

Reply:
nice catch. since I'm updating the test anyway, I've made the change to remove the unnecessary sort.